internal/featuretests/v3/replaceprefix_test.go

```diff
@@ -478,46 +478,45 @@ func artifactoryDocker(t *testing.T) {
 		Resources: resources(t,
 			envoy_v3.RouteConfiguration("ingress_http",
 				envoy_v3.VirtualHost("artifactory.projectcontour.io",
-
 					&envoy_route_v3.Route{
-						Match: routePrefix("/v2/container-sandbox/"),
+						Match: routePrefix("/v2/container-external/"),
```

Comment:
though this makes the order different, that is basically just cosmetic, should still be a no-op change for prefix match routes since the matching mechanism does a lexical/simple string comparison match against what is configured here) (as opposed to a regex match)

Reply:
prefix did similar `lexi` sorting which I guess is also not correct

Reply:
I might be missing the point comment so if I do let me know 😅 But basically prefix routing also had lexical/simple string comparison so I fixed that part as well. Since it is important in both cases, both regexp and prefix. So this is why we needed to change this test

Reply:
oh yeah my comment was just for context for any other readers, to offer some justification for why this change happens in the ordering